app-text/txt2man/txt2man-1.7.1.recipe

```diff
@@ -2,11 +2,11 @@ SUMMARY="Scripts to convert regular ASCII text to man pages"
 DESCRIPTION="Txt2man converts flat ASCII text to man page format. It is a \
 shell script using gnu awk, that should run on any Unix like system."
 HOMEPAGE="https://github.com/mvertes/txt2man"
-COPYRIGHT="2001-2016 Marc Vertes"
+COPYRIGHT="2001-2020 Marc Vertes"
 LICENSE="GNU GPL v2"
 REVISION="1"
 SOURCE_URI="https://github.com/mvertes/txt2man/archive/txt2man-$portVersion.tar.gz"
-CHECKSUM_SHA256="f6939e333a12e1ecceccaa547b58f4bf901a580cd2d8f822f8c292934c920c99"
+CHECKSUM_SHA256="4d9b1bfa2b7a5265b4e5cb3aebc1078323b029aa961b6836d8f96aba6a9e434d"
 SOURCE_DIR="txt2man-txt2man-$portVersion"
 
 ARCHITECTURES="all !x86_gcc2"
```

Comment:
If this is just a shell script, why is this not "any"?

Reply:
Beats me, didn't change that part ;)